src/containers/BooksForm.js

```diff
@@ -10,31 +14,76 @@ const category = [
   { id: 7, name: 'Sci-fi' },
 ];
 
-const BooksForm = () => (
-  <div>
-    <form>
-      <label htmlFor="title">
-        The Title of the Book
-        <input
-          type="text"
-          placeholder="enter the title of the book"
-          name="title"
-          id="title"
-        />
-      </label>
-      <label htmlFor="select">
-        <select>
-          {category.map(c => (
-            <option key={c.id}>{c.name}</option>
-          ))}
-        </select>
-      </label>
-      <button type="submit" value="submit">
-        Submit
-      </button>
-    </form>
-  </div>
-);
-
-
-export default BooksForm;
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = (ev) => {
+    ev.preventDefault();
+    const { value, name } = ev.target;
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit = (ev) => {
+    ev.preventDefault();
+    const { title, category } = this.state;
+    const id = Math.floor(Math.random() * 100);
```

Comment:
[Optional]

You seem to have a function that does this already, `idGen` in `src/index.js`

We could refactor this to be a helper function and import in anywhere it's needed.